integrations/anthropic/integration.definition.ts

```diff
@@ -3,7 +3,7 @@ import { modelId } from 'src/schemas'
 
 export default new IntegrationDefinition({
   name: 'anthropic',
-  version: '0.3.0',
+  version: '1.0.0',
```

Comment:
**note**: Did a major version bump for the 3 AI integrations as the LLM interface has a breaking change (listModels -> listLanguageModels).